keys/README.md

```diff
@@ -0,0 +1 @@
+This folder contains the identity files.
```

Comment:
Maybe to search for key pairs in `~/.ssh/` is the best solution to avoid conflict with paths.